### ISSUE TITLE
Second level menu's broken

### ISSUE THREAD

**Issue by alexratman** [DSH105/SparkTrail#4]
I have version 3.0.1-SNAPSHOT running on Spigot [git-Spigot-1463 (MC:1.7.9) API: 1.7.9-R0.1-SNAPSHOT]

The plugin is working well, interacting perfectly with players and locations. However I have one issue which I have found. 

When I do "/trail" to open the GUI menu I can access all of the trail effects. However if i click on Smoke or Potion (something with a second level menu) and try to configure the color I can only move the item, it does not interact with the plugin, I can move the item into inventory and use the items normally. There is no errors in the console, or in sparktrail.log

I am sure these effects are working however, because if i do "/trail smoke red" it will give me red smoke on the player. It is likely that this bug only occurs on 1.7.9 with the versions I am running, however it may be useful for future updates.

**Comment by mibby** [comment]
@DSH105 
Can confirm that the menu GUI breaks when entering a second window layer. Latest dev build.

**Comment by chaseoes** [comment]
@DSH105
Can also confirm that the GUI is broken in second window layers. This is still an issue.

**Comment by stuntguy3000** [comment]
17 minutes later,

@DSH105
Can also confirm that the GUI is broken in second window layers. This is **still** an issue.

**Comment by chaseoes** [comment]
@DSH105 
Can confirm that this is indeed still an issue.

**Comment by stuntguy3000** [comment]
@DSH105 
Can confirm that this is indeed still an issue.

**Comment by DSH105** [comment]
I'm assuming it's still an issue now too?

**Comment by stuntguy3000** [comment]
@DSH105 
Can confirm that this is indeed still an issue.

**Comment by mibby** [comment]
Q_Q
No sparktrail love. </3

**Comment by mazterredz** [comment]
This is still an issue, can you fix or give a walkaround? Thanks

**Comment by Squawkers13** [comment]
Still issue.

**Comment by chaseoes** [comment]
Confirmed, still an issue.

**Comment by DSH105** [comment]
4 hours later, this must still be an issue, @chaseoes

**Comment by chaseoes** [comment]
Yes, still an issue, @DSH105.

**Comment by mazterredz** [comment]
Are you planning on updating this plugin? @DSH105

**Comment by DSH105** [comment]
Yes, just not right now as I have more important things to deal with.
